lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,38 @@
 module Minesweeper_2pl
   class App
-    def call
-      "Hello World"
+    SIZE = 100
+    BOMB_COUNT = 10
+    attr_accessor :game, :cli
+
+    def start
+      self.setup
+      self.play_game
+    end
+
+    def setup
+      game = Game.new
+      cli = CLI.new
+      self.cli = cli
+      self.game = game
+      self.game.setup(SIZE, BOMB_COUNT)
+    end
+
+    def play_game
+      while self.game.game_over != true
+        self.game.print_board
+        move = nil
+        while move == nil
+          self.cli.ask_for_move
+          move = self.cli.get_player_input(self.game)
+        end
+        coords = self.game.move_to_coordinates(move)
+        self.game.place_move(coords)
+      end
+      if self.game.game_over
+        self.game.show_bombs = true
+        self.game.print_board
```

Comment:
"Reaching into" the `game` object to call methods and access data more than a few times is a little bit of a code smell. Instead of "telling" the `game` what to do and in what order, consider "asking" it for exactly what you need. Here's a great blog post on this idea: http://patshaughnessy.net/2014/2/10/use-an-ask-dont-tell-policy-with-ruby. (There's more in POODR).